engineering_leadership/passed_proposals/readme.md

```diff
@@ -0,0 +1,11 @@
+This is a place to keep track of passed proposals made through engineering leadership channels.
+## Format of Proposals
+***
+#### Date
+12-06-22
+#### Proposal 
```

Comment:
should this include a link to the proposal in slack (if it was made digitally)?

Reply:
it absolutely can, done